File: pr1-cs/Networking/DataTransfer/UserDto.cs
using System;

namespace Networking.DataTransfer
{
    [Serializable]
    public class UserDto
    {
        public UserDto(string username, string password)
        {
            Username = username;
            Password = password;
        }

        public string Username { get; }
        public string Password { get; }

        public override string ToString()
        {
            return "UserDto[" + Username + " " + Password + "]";
        }
    }
}

File: pr1-java/networking/src/main/java/pr1Java/networking/reflection/datatransfer/UserDto.java
package pr1Java.networking.reflection.datatransfer;

import java.io.Serializable;

public class UserDto implements Serializable {
    private final String username;
    private final String password;

    public UserDto(String username, String password) {
        this.username = username;
        this.password = password;
    }

    public String getUsername() {
        return username;
    }

    public String getPassword() {
        return password;
    }

    @Override
    public String toString() {
        return "UserDto[" + getUsername() + " " + getPassword() + "]";
    }
}
